docs/integrations/airflow.md

```diff
@@ -9,30 +9,38 @@ redirect_from: ../using/airflow.html
 ---
 
 # Using lakeFS with Airflow
+
 [Apache Airflow](https://airflow.apache.org/) is a platform to programmatically author, schedule and monitor workflows.
 
-There are some aspects we will need to handle in order to run Airflow with lakeFS:
+There are several steps needed to run Airflow with lakeFS.
+
+## Create a lakeFS connection on Airflow
 
-## Creating the lakeFS connection
-For authenticating to the lakeFS server, you need to create a new [Airflow Connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html)
-of type HTTP and pass it to your DAG. You can do that using the Airflow UI or the cli.
-Here’s an example Airflow command that does just that:
+To access the lakeFS server and authenticate with it, create a new [Airflow
+Connection](https://airflow.apache.org/docs/apache-airflow/stable/howto/connection.html)
+of type HTTP and pass it to your DAG. You can do that using the Airflow UI
+or the cli.  Here’s an example Airflow command that does just that:
 
 ```bash
 airflow connections add conn_lakefs --conn-type=HTTP --conn-host=http://<LAKEFS_ENDPOINT> \
- --conn-extra='{"access_key_id":"<LAKEFS_ACCESS_KEY_ID>","secret_access_key":"<LAKEFS_SECRET_ACCESS_KEY>"}'
+    --conn-extra='{"access_key_id":"<LAKEFS_ACCESS_KEY_ID>","secret_access_key":"<LAKEFS_SECRET_ACCESS_KEY>"}'
 ```
 
-## Installing lakeFS Airflow package
-Installing the package using `pip`:
```

Comment:
please add the package link - https://github.com/treeverse/airflow-provider-lakeFS

Reply:
Added a package link; I think it is https://pypi.org/project/airflow-provider-lakefs/ so I added that link.  Users should care less about where the package is built, and more about from where to download it.

Reply:
good point!